src/views/admin/PendingCredits.vue

```diff
@@ -103,10 +123,14 @@
       },
     },
     methods: {
-      async validateCreditRequest(request: any): Promise<void> {
+      async validateCreditRequest(request: any, index: any): Promise<void> {
+        this.selectedItem = index
```

Comment:
You don't need to use the index to store the `selectedItem`. Just do `this.selectedItem = request` and in your template avoir the index, and test with `selectedItem !== request`.

Reply:
you are right, thank you for the review